### PR TITLE
Make Email Button Gradient Highlight Consistent with Text Boxes

### LINKED ISSUE #48: Contact Send Email Button Blue Hover Gradient
Change Send Email bottom gradient to the ones used by the input text boxes

### PR DESCRIPTION
**Pull request by byue**
Made Highlight colors consistent among all text boxes and buttons. Highlight band was also made thinner. Text Input boxes made thinner. Max Text Length was added.

### CODE REVIEW

**Comment by byue** [comment]
Screenshot (1)

**Comment by byue** [comment]
Before: 
Screenshot (4)

File: src/components/ui/button.tsx [hunk 1]
```diff
@@ -0,0 +1,61 @@
+// Input component extends from shadcnui - https://ui.shadcn.com/docs/components/input
+"use client";
+import * as React from "react";
+import { cn } from "../../lib/utils";
+import { useMotionTemplate, useMotionValue, motion } from "framer-motion";
+
+export interface ButtonProps
+  extends React.ButtonHTMLAttributes<HTMLButtonElement> {}
+
+const Button = React.forwardRef<HTMLButtonElement, ButtonProps>(
```
Comment (bryue) [inline src/components/ui/button.tsx]: More specific name if this button is only used for form submission

File: src/pages/index.tsx [hunk 2]
```diff
@@ -60,15 +61,6 @@ const LabelInputContainer = ({
   );
 };
 
-const BottomGradient = () => {
```
Comment (bryue) [inline src/pages/index.tsx:63]: Why was this button specifically coded to have different color gradient from the rest of the form?

**Review by bryue — CHANGES_REQUESTED** [review]

**Comment by byue** [comment]
After:
Screenshot (5)

**Comment by byue** [comment]
After:
Screenshot (6)

**Comment by byue** [comment]
Before:
image

**Comment by byue** [comment]
Before: 
Screenshot (8)

**Review by bryue — APPROVED** [review]

**Pull request merged**